src/app/about/about.component.css

```diff
@@ -1 +1,5 @@
-
+img.dev-cycle {
+      display: block;
+      margin-left: auto;
+      margin-right: auto
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a semicolon at the end of this declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=css%3Asemicolon-declaration)